planning/kinematics_plugin_loader/src/kinematics_plugin_loader.cpp

```diff
@@ -76,6 +76,11 @@ class KinematicsPluginLoader::KinematicsLoaderImpl
     }
   }
 
+  ~KinematicsLoaderImpl( ) {
+      kinematics_loader_.reset(); // we delete it manually
+      ROS_DEBUG( "KinematicsLoaderImpl destructor");
+  }
+
```

Comment:
This commit has nothing to do with the matter addressed in the pull-request and is separately discussed in #702.
Please remove this commit from the request.